benchmark/CMakeLists.txt

```diff
@@ -213,7 +216,9 @@ set(SWIFT_BENCH_MODULES
     cxx-source/ReadAccessor
 )
 
-set(SWIFT_MULTISOURCE_SWIFT_BENCHES
+set(SWIFT_MULTISOURCE_SWIFT_BENCHES)
+# FIXME(NCG): Temporarily disabled 
+set(SWIFT_MULTISOURCE_SWIFT_BENCHES_DISABLED)
   multi-source/PrimsSplit
```

Comment:
Extraneous trailing `)` on the line above is causing this error:
```
CMake Error at benchmark/CMakeLists.txt:222:
  Parse error.  Expected a command name, got unquoted argument with text
  "multi-source/PrimsSplit".
```

Reply:
Wow, that was silly! I removed all these changes, reenabling benchmarking. I'm running a local build to see if anything remains broken.